python/grpc/hello_pb2_grpc.py

```diff
@@ -0,0 +1,46 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
+import grpc
+
+import hello_pb2 as hello__pb2
+
+
+class HelloStub(object):
+  # missing associated documentation comment in .proto file
+  pass
+
+  def __init__(self, channel):
+    """Constructor.
+
+    Args:
+      channel: A grpc.Channel.
+    """
+    self.SayHello = channel.unary_unary(
+        '/Hello/SayHello',
+        request_serializer=hello__pb2.HelloRequest.SerializeToString,
+        response_deserializer=hello__pb2.HelloReply.FromString,
+        )
+
+
+class HelloServicer(object):
+  # missing associated documentation comment in .proto file
+  pass
```

Comment:
This `pass` shouldn't be needed.

It would be if there was no comment or `def SayHello` was not part of the object.

Reply:
this is in a generated file, sorry